### DISCUSSION TITLE
Auto-read articles older than 30 days

### DISCUSSION THREAD

**Discussion by CNote1** [FreshRSS/FreshRSS#6704] [Q&A]
How can mark-as-read articles older than 30 days?

**Comment by math-GH** [comment]
You could use the search field to filter the older articles and then mark them as read

![grafik](https://github.com/user-attachments/assets/079ed657-f578-4e7c-8608-3118a790d1cf)
https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field
